### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by aeftink** [codeofsumit/leaflet.pm#262]
When working with GeoJSON, I get a maximum call stack size exceeded error.
JS Fiddle: https://jsfiddle.net/alexeftink/Lh31ujty/

Scenario
I have a feature group of GeoJSON objects.  When a user edits one of those objects, they may want to revert their changes.  To do that I save the original feature from the geojson layer.  Then when they cancel their edits I call layer.clearLayers() and layer.addData(feature).  However, that results in maximum call stack size exceeded.  Looks like the enabled() function ends up calling itself recursively indefinitely.

**Comment by codeofsumit** [comment]
Thanks for the fiddle! Mhh, it does have something to do with nested feature/layerGroups. I tried to find the cause for the past 2 hours to no avail. Will get back to it soon!

**Comment by aeftink** [comment]
Yeah I haven't figured it out yet either.  Its as if the _layers in the LayerGroup somehow contains an instance of itself
